### PR TITLE
Indent "Hello" With Tabs

### PR DESCRIPTION
**Pull request by minimaxir**
Use tabs for indenting salutations.

### CODE REVIEW

**Comment by jpotisch** [comment]
👍🏻

**Comment by mrdoob** [comment]
LGTM

File: Hello [hunk 1]
```diff
@@ -1 +1 @@
-Hi
+  Hi
```
Comment (robert-w-gries) [inline Hello:1]: Use 8 spaces instead of 2

``` diff
-  Hi
+        Hi
```

Comment (minimaxir) [inline Hello:1]: I do not negotiate with terrorists.

**Comment by VaughnGH** [comment]
Honestly, this is an outrage

**Comment by christimahu** [comment]
Please perform middle out rebase before merging.

**Comment by crk042** [comment]
This format style offends me.

File: Hello (hunk 1 above)
Comment (lewismoten) [inline Hello:1]: You need an opening block before you start indenting.

```
Dear Mum,
    Hi
```

**Comment by camargo** [comment]
I hope this was done with VIM

**Comment by hatsunearu** [comment]
so is she a spy or something? why is her github full of fake commits?

**Comment by lewismoten** [comment]
@hatsunearu - It's an account for a character on a TV show. The fake commits are to make her account to appear active.

**Comment by YashalKanungo** [comment]
But its all the same once compiled.

**Comment by brendonjohn** [comment]
👍

**Comment by rhoit** [comment]
not if you are doing python! :snake:

**Comment by esr360** [comment]
To be honest I myself am a man of 4 space indentation.

**Comment by hatsunearu** [comment]
I get that but maybe her character is a spy and the fake commits are in-universe?

On May 29, 2016 11:40 PM, Lewie notifications@github.com wrote:

@hatsunearuhttps://github.com/hatsunearu - It's an account for a character on a TV show. The fake commits are to make her account to appear active.

**Comment by kz** [comment]
Definitely not. We should be using spaces.

**Comment by 0xcaff** [comment]
It would be more efficient to use vim's `:set expandtab`, `:set tabstop=8` and `:set shiftwidth=8`

**Comment by KonradIT** [comment]
And we should be using Vim, because its vastly superior

**Comment by thesourabh** [comment]
I cannot in good conscience remain part of a project that uses tabs instead of spaces. This is monstrous.

**Comment by justjoeyuk** [comment]
This is amazing haha!

**Comment by olzaragoza** [comment]
According to Richard Hendricks, CEO of [Pied Piper](http://www.piedpiper.com/):

> "Tabs create smaller file sizes, alright.  I run a compression company.  Trust me I've devoted my life to minimalizing file sizes.  It's what I do.  I mean, I do not get why anyone would use spaces over tabs.  I mean, why not just use Vim over Emacs."

**Comment by svennidal** [comment]
People who use spaces instead of tabs are the same people that hold your hand for too long when they are greeting you. 

Save your keystrokes!

**Comment by mrdoob** [comment]
MERGED?! SPOILER ALERT!

**Comment by minimaxir** [comment]
TABS PREVAIL

**Comment by mazz** [comment]
spaces rule

**Comment by esr360** [comment]
You know you're in the right job when you see a spoiler for your favourite TV show whilst doing work

**Comment by justjoeyuk** [comment]
OH MY LORD, IT WORKS OUT

**Comment by Enelar** [comment]
@olzaragoza actually on `better file system`s it doesn't matter. As they compressing files on the fly. Richard tried to brag. Also why he is not on github?

**Comment by dashed** [comment]
Hi mom. 👋

**Comment by justjoeyuk** [comment]
I'm calling it. She's returning this season and will concede.

**Pull request merged**